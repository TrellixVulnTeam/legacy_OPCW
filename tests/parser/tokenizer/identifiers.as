abcd32
asfs
__sfs
