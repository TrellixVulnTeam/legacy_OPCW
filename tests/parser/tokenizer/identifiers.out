<identifier> 'abcd32'
<identifier> 'asfs'
<identifier> '__sfs'
<end>
